apps/seqan_flexbar/seqan_flexlib.cpp

```diff
@@ -257,11 +271,17 @@ void ArgumentParserBuilder::addAdapterTrimmingOptions(seqan::ArgumentParser & pa
         "np", "no-paired", "Trim paired-end input with single-end trimming method.");
     addOption(parser, pairedModeOpt);
 
-    seqan::ArgParseOption rateOpt = seqan::ArgParseOption(
+    seqan::ArgParseOption errorOpt = seqan::ArgParseOption(
```

Comment:
yes, because the old e parameter was actually about absolute errors, so i left the function the same and just changed the description of it.